packages/cfd/src/Stores/Modules/CFD/cfd-store.js

```diff
@@ -373,16 +380,19 @@ export default class CFDStore extends BaseStore {
                 WS.transferBetweenAccounts();
                 this.root_store.client.responseMT5TradingServers(await WS.tradingServers(CFD_PLATFORMS.MT5));
             } else {
+                actions.setStatus({ success: false });
                 await this.getAccountStatus(CFD_PLATFORMS.MT5);
                 this.clearCFDError();
                 this.setMT5MigrationError(has_error?.error?.message);
                 this.setMigratedMT5Accounts([]);
```

Comment:
why do we remove toggling of migrationmodal?

Reply:
as per the latest figma flow, we don't have to `go back` to the migration modal. we will be having password field within the migration modal itself. so toggling is done from the `MT5MigrationBackSideContent` component.